lib/carrierwave/mounter.rb

```diff
@@ -116,7 +116,7 @@ def blank?
     end
 
     def remove?
-      remove.present? && (remove == true || remove !~ /\A0|false$\z/)
+      remove.present? && (remove == true || remove.to_s !~ /\A0|false$\z/)
```

Comment:
```suggestion
      remove.present? && (remove.to_s !~ /\A0|false$\z/)
```

We can drop `remove == true` , right?